Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM mitmproxy/mitmproxy:3.0.3
 EXPOSE 8081
 COPY ./ /code
 COPY ./mitmproxy-ca.pem /root/.mitmproxy/mitmproxy-ca.pem
-RUN apk add --update py3-pip python3-dev gcc musl-dev
+RUN apk update && apk add --update py3-pip python3-dev gcc musl-dev
```

Comment:
Shouldn't be very useful according to https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#packages